skbio/stats/composition.py

```diff
@@ -608,7 +613,12 @@ def ilr_inv(mat, basis=None, check=True):
 
     check: bool
         Specifies if the basis is orthonormal.
-
+    
+    Returns
+    -------
+    np.array
+        Inverse ilr coordinates representing a D dimensional
```

Comment:
"representing a D dimensional" -> "representing D dimensional"

should "ilr" be capitalized?